content/en/docs/concepts/services-networking/service.md

```diff
@@ -384,6 +384,33 @@ The IP address that you choose must be a valid IPv4 or IPv6 address from within
 If you try to create a Service with an invalid clusterIP address value, the API
 server will return a 422 HTTP status code to indicate that there's a problem.
 
+## Traffic Policies
+
+### External Traffic Policy
+
+You can set the `spec.externalTrafficPolicy` field to control how traffic from external sources is routed.
+Valid values are `Cluster` and `Local`. Set the field to `Cluster` to route external traffic to all ready endpoints
+and `Local` to only route to ready node-local endpoints. If the traffic policy is `Local` and there are are no node-local
+endpoints, traffic is dropped by kube-proxy.
```

Comment:
```suggestion
endpoints, the kube-proxy does not forward any traffic for the relevant Service.
```
I presume the exact behavior in this case depends on whether you're using kube-proxy in ipvs, iptables or userland mode. Is that right?